src/dialogs/more-info/controls/more-info-light.ts

```diff
@@ -66,10 +66,11 @@ class MoreInfoLight extends LitElement {
               <ha-labeled-slider
                 caption=${this.hass.localize("ui.card.light.brightness")}
                 icon="hass:brightness-5"
-                min="1"
-                max="255"
+                min="0"
+                max="100"
```

Comment:
By changing this the precision of the slider is now 2.55 times less

Reply:
+ we now have rounding differences, not sure those are issues but something to consider

Reply:
True, but is that something a user needs? Having the 0..100 percent is much more user friendly, plus it mirrors what we already use for the rounded slider in the hui-light-card.

Reply:
The hui-light-card however for contrast is only dividing by 254 before Math.round() compared that the 255 what I used now. Change made by you some time ago (#4257).

Reply:
I didn't change the logic there, just moved the line ;-)

Reply:
The 254 is from the original light card <https://github.com/home-assistant/frontend/pull/1874>